metrics/web/lib/common/presentation/routes/route_generator.dart

```diff
@@ -48,15 +49,15 @@ class RouteGenerator {
     );
   }
 
-  /// Creates [MaterialPageRoute] with the given [name] and the [widget].
+  /// Creates [MetricsPageRoute] with the given [name] and the [widget].
   ///
   /// Throws an [AssertionError] if the [widget] is null.
   static MaterialPageRoute _createMaterialPageRoute({
```

Comment:
Should we rename this method and make it return MetricsPageRoute? 
```suggestion
  static MetricsPageRoute _createMetricsPageRoute({
```